presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -45,6 +45,9 @@ Property Name                          Description
 ``iceberg.catalog.cached-catalog-num`` The number of Iceberg catalogs to cache.
 
 ``iceberg.hadoop.config.resources``    The path(s) for Hadoop configuration resources.
+
```

Comment:
Can we also cherry pick and improve the rest of the property documentation?
For example, for `iceberg.file-format`, can we also add something like "Available values are``PARQUET``, ``ORC``, defaults to ``ORC ``". Same for the rest of the configs

Reply:
> Can we also cherry pick and improve the rest of the property documentation? For example, for `iceberg.file-format`, can we also add something like "Available values are`PARQUET`, `ORC`, defaults to `ORC `". Same for the rest of the configs

Presto already has that information.

Reply:
It is in the doc, please see line #63